runbotics/runbotics-scheduler/src/websocket/bot/process-launch/bot-process-instance-event.service.ts

```diff
@@ -177,10 +192,18 @@ export class BotProcessEventService {
             await queryRunner.manager
                 .createQueryBuilder()
                 .update(ProcessInstanceEntity)
-                .set({
-                    error: processInstanceEvent.error,
-                    step: processInstanceEvent.step,
-                })
+                .set(
+                    setWarning(processInstanceEvent)
+                        ? {
```

Comment:
manipulate just a warning field with this function
`
{
    error: processInstanceEvent.error,
    step: processInstanceEvent.step,
    warning: hasWarning(processInstanceEvent.step),
}
`